packages/aragon-cli/src/middleware/environment.js

```diff
@@ -178,20 +144,12 @@ module.exports = function environmentMiddleware(argv) {
       `Could not find 'arapp.json'. Using the default configuration to connect to ${env.network}.`
     )
 
-    // Override apm options that we find in the environment
-    // TODO (daniel) : it should be the other way around though (cli params to override env)
-    if (apm) {
-      if (env.registry) {
-        apm['ens-registry'] = env.registry
-      }
-      if (env.apm) {
-        apm = merge(apm, env.apm)
-      }
-    }
-
     return {
-      // TODO (daniel) : remove this as it does not seem to be used
-      apmEnsRegistry: env.registry,
+      apm: configureAPM(
+        env.apm.ipfs.rpc || argv['ipfs-rpc'],
```

Comment:
Same as https://github.com/aragon/aragon-cli/pull/895/files#diff-7fb765944652f71e14673b01c4e751d9R118

Can you guarantee that env.apm is defined and is an object in this line?